### ISSUE TITLE
Review prey lists

### ISSUE THREAD

**Issue by slucey** [NOAA-EDAB/benthosindex#6]
Ensure that species/groups are assigned correctly to mega and micro benthos.

**Comment by SarahJWeisberg** [comment]
Made decision to move PECTINIDAE, PECTINIDAE SHELL, PECTINIDAE VISCERA to AtlScallop group.

**Comment by SarahJWeisberg** [comment]
Only other thing of note is that mole / sand crabs (eg., ALBUNEA PARETII, EMERITA TALPOIDA, RANILIA MURICATA, HIPPIDAE) are currently in the megabenthos category. Might make sense to move them to macrobenthos. Probably not a big deal either way, as these are relatively rare (ALBUNEA PARETII found in 35 stomachs, rest in fewer).

**Comment by sgaichas** [comment]
I agree that mole/sand crabs might be better off in macrobenthos. This may make a difference in the NEAMAP data closer to shore, so I'll go ahead and make that change if no one objects.

**Comment by SarahJWeisberg** [comment]
Sounds good! Thanks Sarah
